### ISSUE TITLE
Waiting mercenaries start to roam after world restart

### ISSUE THREAD

**Issue by Tholwin** [jpw1991/chebs-mercenaries#57]
When I restart the world, mercenaries that were waiting are walking around, even though their status is still "waiting".

I can't find any error in the logs.

**Comment by jpw1991** [comment]
Yeah, basically the same problem as your other issue [here](https://github.com/jpw1991/chebs-mercenaries/issues/56). I'll look into it soon

**Comment by Tholwin** [comment]
Sorry for the double post. At first, I didn't realize it was a bug.
